Mathlib/Algebra/Operad/Operad.lean

```diff
@@ -0,0 +1,136 @@
+/-
```

Comment:
This file should be renamed `Defs`.

Reply:
(and I guess `Basic` needs renaming too if that is even more basic that this file!)